.github/pull_request_template.md

```diff
@@ -1,44 +1,5 @@
-<!-- Thank you for submitting a pull request! Find more information in our development guide at https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md and contact us on #dev in Slack. -->
```

Comment:
I'd probably keep this.

Reply:
Github has a builtin support for this:

<img width="350" alt="image" src="https://user-images.githubusercontent.com/144328/187413134-0f01c5b9-601f-4952-b36e-adeafd5e18ba.png">

Reply:
(screenshot from some other project)

Reply:
Does that actual work in the Trino project or does it need some specific setup @findepi ?

Reply:
It works automatically if you have a CONTRIBUTING.md for first time submitters.